curriculum/challenges/english/25-front-end-development/workshop-recipe-ingredient-converter/67351754cc06dc00d4661745.md

```diff
@@ -0,0 +1,173 @@
+---
+id: 67351754cc06dc00d4661745
+title: Step 1
+challengeType: 0
+dashedName: step-1
+demoType: onLoad
+---
+
+# --description--
+
+In this workshop, you will apply the functional programming concepts you have learned in the previous lectures to build an ingredient converter app.
+
+Start by declaring an empty `conversationTable` object.
+
+# --hints--
+
+You should use `const` to declare a `conversionTable` variable.
+
+```js
+assert.match(code, /const\s+conversionTable/);
+```
```

Comment:
For here we can be more open ended and allow for `let`, `var` or `const`. Especially, since the use of `const` wasn't explicitly mentioned in the directions. 

probably a `isDefined` should be fine here